docs/Introduction/getting-started/conceptual-overview.md

```diff
@@ -0,0 +1,176 @@
+
+In `HelloWorld`, the constructor takes in a `string` as a parameter and sets the `message` state variable to that string.
+
+```solidity
+constructor(string memory initialMessage) {
+  message = initialMessage;
+}
+```
+
+### Functions
+
+**Functions** are used to access and modify the state of the contract, and call functions on external contracts. `HelloWorld` has a function called `updateMessage`, which updates the current message stored in the state.
+
+```solidity
+constructor(string memory initialMessage) {
+  message = initialMessage;
+}
+
+function updateMessage(string memory newMessage) public {
+  message = newMessage;
+}
+```
+
+### Interfaces
+
+An **interface** is another concept that is familiar to programmers of other languages. Interfaces define functions without their implementation, which leaves inheriting contracts to define the actual implementation themselves. This makes it easier to know what functions to call in a contract. Here's an example of an interface:
+
+```solidity
+{% include samples/Tutorials/Test.sol %}
+```
+<div class="remix-callout">
+  <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/Tutorials/Test.sol" target="_blank">Deploy using Remix</a>
+  <a href="/docs/conceptual-overview/#what-is-remix" >What is Remix?</a>
+</div>
+
+For this example, `override` is necessary in the `Test` contract function because it overrides the base function contained in the `numberComparison` interface. The contract uses `pure` instead of `view` because the `isSameNum` function in the `Test` contract does not return a storage variable.
+
+## What does "deploying" mean?
+
+**Deploying** a smart contract is the process of pushing the code to the blockchain, at which point it resides with an on-chain address. Once it's deployed, the code cannot be changed and is said to be *immutable*.
+
+As long as the address is known, its functions can be called through an interface, on [Etherscan](https://etherscan.io/), or through a library like [web3js](https://web3js.readthedocs.io/), [web3py](https://web3py.readthedocs.io/), [ethers](https://docs.ethers.io), and more. Contracts can also be written to interact with other contracts on the blockchain.
+
+## What are oracles?
+
+**Oracles** provide a bridge between the real-world and on-chain smart contracts, by being a source of data that smart contracts can rely on, and act upon.
+
+Oracles play an extremely important role in facilitating the full potential of smart contract utility. Without a reliable connection to real-world conditions, smart contracts are unable to effectively serve the real-world.
+
+## How do smart contracts use oracles?
+
+Oracles are most popularly used with [*Data Feeds*](../using-chainlink-reference-contracts/). DeFi platforms like [AAVE](https://aave.com/) and [Synthetix](https://www.synthetix.io/) use Chainlink data feed oracles to obtain accurate real-time asset prices in their smart contracts.
```

Comment:
I think we need to mention all of our products, or talk more generally about Oracles giving you access to data and capabilities.

Reply:
Added all of our products to this section with descriptions.